packages/nouns-webapp/src/components/Auction/index.tsx

```diff
@@ -43,16 +43,22 @@ const Auction: React.FC<AuctionProps> = props => {
     currentAuction && history.push(`/noun/${currentAuction.nounId.toNumber() + 1}`);
   };
 
-  const handleKeyPress = useCallback(event => {
-    if (event.code === 'ArrowLeft') prevAuctionHandler();
-    else if (event.code === 'ArrowRight') nextAuctionHandler();
-    else return;
-  }, []);
+  const handleKeyPress = (event: KeyboardEvent) => {
```

Comment:
The current code will cause adding and removing the event listener on every render because this function is being re-created on each render. Take note of the compiler warning about this:
```
src/components/Auction/index.tsx
  Line 46:9:  The 'handleKeyPress' function makes the dependencies of useEffect Hook (at line 61) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'handleKeyPress' in its own useCallback() Hook
```